languages/nickel.scm

```diff
@@ -0,0 +1,193 @@
+; Configuration
+(#language! nickel)
+
+[
+  (static_string)
+  (str_chunks_single)
+] @leaf
```

Comment:
For consistency we should add the same comment here about `leaf` that all the other query files include.